operator/builtin/input/stanza/stanza.go

```diff
@@ -0,0 +1,132 @@
+package stanza
+
+import (
+	"context"
+	"sync"
+
+	"github.com/observiq/stanza/entry"
+	"github.com/observiq/stanza/logger"
+	"github.com/observiq/stanza/operator"
+	"github.com/observiq/stanza/operator/helper"
+	"go.uber.org/zap/zapcore"
+)
+
+func init() {
+	operator.Register("stanza_input", func() operator.Builder { return NewInputConfig("") })
+}
+
+// NewInputConfig creates a new stanza input config with default values
+func NewInputConfig(operatorID string) *InputConfig {
+	return &InputConfig{
+		InputConfig: helper.NewInputConfig(operatorID, "stanza_input"),
+		BufferSize:  100,
+	}
+}
+
+// InputConfig is the configuration of a stanza input operator.
+type InputConfig struct {
+	helper.InputConfig `yaml:",inline"`
+	BufferSize         int `json:"buffer_size" yaml:"buffer_size"`
+}
+
+// Build will build a stanza input operator.
+func (c *InputConfig) Build(context operator.BuildContext) (operator.Operator, error) {
+	inputOperator, err := c.InputConfig.Build(context)
+	if err != nil {
+		return nil, err
+	}
+
+	receiver := make(logger.Receiver, c.BufferSize)
+	context.Logger.AddReceiver(receiver)
```

Comment:
It looks like order matters when building. If we build another operator before building this operator, and it fails, we won't see that log. Seems like it might be difficult to get around that, but was wondering if you'd thought about that at all